### ISSUE TITLE
VkRenderingInfo document error in 1.3.243

### ISSUE THREAD

**Issue by FuXiii** [KhronosGroup/Vulkan-Docs#2078]
When I reading the [Vulkan-Doc 1.3.243:VkRenderingInfo](https://registry.khronos.org/vulkan/specs/1.3-khr-extensions/html/chap8.html#VkRenderingInfo)

![image](https://user-images.githubusercontent.com/40132375/225179554-fab72390-bcf7-4c7b-bd67-8b28c00bc3ef.png)

I can not found the member named `deviceCount` in `VkDeviceGroupRenderPassBeginInfo`

```CXX
// Provided by VK_VERSION_1_1
typedef struct VkDeviceGroupRenderPassBeginInfo {
    VkStructureType    sType;
    const void*        pNext;
    uint32_t           deviceMask;
    uint32_t           deviceRenderAreaCount;
    const VkRect2D*    pDeviceRenderAreas;
} VkDeviceGroupRenderPassBeginInfo;
```

**Comment by Tobski** [comment]
Ah simple mistake, it's supposed to be "deviceRenderAreaCount". I'll raise an MR to fix it.

**Comment by FuXiii** [comment]
Ok，thanks~

**Comment by oddhack** [comment]
This is fixed in the 1.3.244 spec update.
